Add resizable thumbnail interface (#2040)
sonata-project/SonataMediaBundle: UPGRADE-3.x.md

UPGRADE 3.x
===========

UPGRADE FROM 3.x to 3.x
=======================

> ### ResizableThumbnailInterface

Reviewer: ```suggestion
### Sonata\MediaBundle\Thumbnail\ResizableThumbnailInterface
```

> This interface is added to provide a clearer Api for FormatThumbnail. It allows to know if a Thumbnail can be generated with custom Resizers, not the ones provided by

Reviewer: ```suggestion
This interface is added to provide a clearer API for `FormatThumbnail`. It allows to know if a `Thumbnail` object can be generated with custom `Resizer` instances, not the ones provided by
```

the file provider.

> If you have custom Thumbnail generator, please implement this interface if you need

Reviewer: ```suggestion
If you have custom `Thumbnail` generator, please implement this interface if you need
```
